tutorials/fundamentals/typeof-undefined.md

```diff
@@ -0,0 +1,16 @@
+To check if a variable is undefined, you should use
+the [`typeof` operator](/tutorials/fundamentals/typeof).
+When used on an `undefined` variable, it will return
+`'undefined'`. If you use that in conjunction with the
+`===` operator, you can successfully check for those specific
+values.
+
+```javascript
+var x;
+typeof x; // 'undefined'
+if (x === 'undefined') {
+    // will execute
+}
+```
+
+**Note:** Read more about checking for [`undefined`](/tutorials/fundamentals/undefined-check).
```

Comment:
Make the "check if a variable is undefined" link to the `/tutorials/fundamentals/undefined-check` tutorial, and remove this bit.